src/types.ts

```diff
@@ -59,6 +59,7 @@ export interface PostHogConfig {
     api_host: string
     api_method: string
     api_transport: string
+    asset_host?: string
     ui_host: string | null
```

Comment:
```suggestion
    /**
     * PostHog web app,
     * the original web app host needs to be passed here so that links to the web app are still convenient.
    **/
    ui_host: string | null
```